{{cookiecutter.project_name}}/.github/workflows/test_on_push.yml

```diff
@@ -0,0 +1,90 @@
+name: Project Tests
```

Comment:
This file, too, can benefit from the use of a single package management solution as noted above.